keycloak/keycloak-ui
# add localization keys for idp review profile on first login

## Motivation
Fixes #3707 

This needs to be merged at the same time with the related PR in the keycloak main repository: https://github.com/keycloak/keycloak/pull/15293

## Brief Description
In Keycloak admin, navigate to Authentication -> first broker login -> and click on "settings"  on the review profile step. You will see the folliowing:

![0 0 0 0_8080_admin_master_console_](https://user-images.githubusercontent.com/6265563/199524372-aca099d1-4bac-419a-a729-ed1bb3a25ccb.png)


It seems like AngularJs interpolation is not working (anymore). 

## Verification Steps

All you need to do is navigate to Authentication -> first broker login -> and click on "settings"  on the review profile step, and make sure the labels and tooltips are displayed properly. 

## Checklist:


## Additional Notes
